libs/blocks/section-metadata/section-metadata.css

```diff
@@ -183,8 +183,14 @@
   gap: var(--spacing-m);
 }
 
+.section.container[class*='-up'] {
+  width: auto;
+  margin: 0;
+}
+
 .section[class*='grid-width-'] > .content,
-main > .section[class*='-up'] > .content {
+main > .section[class*='-up'] > .content,
+.section[class*='grid-width-'] > .contained {
   max-width: initial;
```

Comment:
unfortunately, this now overrides `max-width: var(--grid-container-width);` coming from [columns.css](https://github.com/adobecom/milo/blob/694389f75d3b1a5e01416f2f771a6246bb27102f/libs/blocks/columns/columns.css#L8) causing a bit of a margin error on mobile:
![current mobile grid](https://github.com/adobecom/milo/assets/17607967/1de83d39-3dd9-490b-bfd0-abac474ac725)
![fix mobile grid](https://github.com/adobecom/milo/assets/17607967/82073c7f-d421-4353-a397-9d4d0857b7b5)